sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2081,6 +2081,22 @@ object functions {
    */
   def size(column: Column): Column = Size(column.expr)
 
+  /**
+   * Returns true if the array contain the value
+   * @group collection_funcs
+   * @since 1.5.0
+   */
+  def array_contains(columnName: String, value: Any): Column =
```

Comment:
It's not a big deal, I think we will do another clean up for the DF APIs. 

@rxin did clean up something already. see https://github.com/apache/spark/pull/7556